tests/test_solution.py

```diff
@@ -779,3 +803,8 @@ def test_to_from_file(tmp_path, s1):
         s1.to_file(filename)
     with pytest.raises(FileNotFoundError, match=r"File .* not found!"):
         Solution.from_file(filename)
+    sol = Solution({"Ca+2": "1 ppt"})
+    ppt_value = sol.get_amount("Ca+2", "ppt").magnitude
+    nanogram_per_liter = sol.get_amount("Ca+2", "nanogram/L").magnitude
+    assert np.isclose(ppt_value, 1), f"Expected 1, got {ppt_value}"
+    assert np.isclose(nanogram_per_liter, 1), f"Expected 1, got {nanogram_per_liter}"
```

Comment:
Looking great @NikhilDhruv ! I think these lines got stuck in the wrong test when you copied them in -- they should be part of `test_unit_conversions` rather than `test_to_from_file`. Please move them up to ~line 790.